src/.nuget/packages.config

```diff
@@ -1,6 +1,7 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Tpl.Dataflow" version="4.5.24" targetFramework="net451" />
   <package id="NUnit" version="2.6.4" targetFramework="net451" userInstalled="true" />
   <package id="Microsoft.Net.ToolsetCompilers" version="1.0.0-rc3-20150510-01" targetFramework="net451" userInstalled="true" />
+  <package id="Microsoft.Net.ToolsetCompilers" version="1.0.0-rc2-20150415-05" targetFramework="net451" userInstalled="true" />
```

Comment:
This is incorrect, we shouldn't have multiple versions of the same package here.